### PR TITLE
8.0 account move line purchase info

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This module will add the purchase order line to invoice lines and to account
 move lines. The ultimate goal is to establish the purchase order line as
 one of the key fields to reconcile the Goods Received Not Invoiced accrual
 account.
# Usage

The purchase order line will be automatically copied to invoice lines and
account move lines.
- If an invoice is created from a purchase order, the PO line is copied
  directly to the invoice, and to the account move line upon validation.
- If an invoice is created from a stock picking, the PO line is copied from
  the stock move to the invoice, and to the account move line upon validation.
- When a stock move is validated and generates an account move, the PO line is
  copied to the account move line.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@elicoidal @pedrobaeza Now added module 'account_move_line_purchase_info' here. This time extensive automated tests have been added.

File: account_move_line_purchase_info/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+===========================
+Stock Account Purchase Line
+===========================
+
+This module will add the purchase order line to invoice lines and to account
+ move lines. The ultimate goal is to establish the purchase order line as
```
Comment (pedrobaeza) [inline account_move_line_purchase_info/README.rst]: don't put an initial space in these lines

File: account_move_line_purchase_info/README.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ one of the key fields to reconcile the Goods Received Not Invoiced accrual
+ account.
+
+
+Usage
+=====
+The purchase order line will be automatically copied to invoice lines and
+account move lines.
+* If an invoice is created from a purchase order, the PO line is copied
+directly to the invoice, and to the account move line upon validation.
```
Comment (pedrobaeza) [inline account_move_line_purchase_info/README.rst]: Indent this with 2 spaces

File: account_move_line_purchase_info/README.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+This module will add the purchase order line to invoice lines and to account
+ move lines. The ultimate goal is to establish the purchase order line as
+ one of the key fields to reconcile the Goods Received Not Invoiced accrual
+ account.
+
+
+Usage
+=====
+The purchase order line will be automatically copied to invoice lines and
+account move lines.
```
Comment (pedrobaeza) [inline account_move_line_purchase_info/README.rst:19]: Let here a line

**Comment by pedrobaeza** [comment]
When formatting README, please see the preview option in GitHub to know the errors you have in the syntax.

**Comment by JordiBForgeFlow** [comment]
Runbot raises this error: 
except_orm: ('Error', u"You try to install module 'account_move_line_purchase_info' that depends on module 'account_invoice_merge_purchase'.\nBut the latter module is not available in your system.")

I guess that the purchase-workflow repo should add 'account-invoice' as a dependency, right?

**Comment by pedrobaeza** [comment]
Why do you need that module? It's very strange that you need a glue module that fixes the purchase reference when merging invoices...

**Comment by JordiBForgeFlow** [comment]
The tests performed on this module fail due to module "stock_ownership_availability_rules" (installed automatically as a dependency on runbot) failing to create account moves. Reported issue here: https://github.com/OCA/stock-logistics-workflow/issues/197

**Comment by JordiBForgeFlow** [comment]
@pedrobaeza I removed dependency module "account_invoice_merge_purchase". I cannot get rid of the  Travis error "README.rst:22 Bullet list ends without a blank line; unexpected unindent."...

**Comment by jgrandguillaume** [comment]
HI,

Thanks for this contribution. LGTM :+1:  once travis green.

Regards,

Joël

**Pull request closed without merging**